app/api/events.py

```diff
@@ -381,7 +381,7 @@ class EventRelationship(ResourceRelationship):
 
     def before_get_object(self, view_kwargs):
         if view_kwargs.get('identifier'):
-            event = self.session.query(Event).filter_by(identifier=view_kwargs['identifier']).one()
+            event = safe_query(self, Event, 'identifier', view_kwargs['identifier'], 'identifier')
             view_kwargs['id'] = event.id
```

Comment:
Well, then we have to modify all the relationship endpoints in __init__.py. I thought we can make those changes in another issue.